src/plugins/intel_gpu/src/kernel_selector/kernels/arg_max_min/arg_max_min_kernel_axis.cpp

```diff
@@ -125,7 +166,18 @@ KernelsPriority ArgMaxMinKernelAxis::GetKernelsPriority(const Params& /*params*/
 JitConstants ArgMaxMinKernelAxis::GetJitConstants(const arg_max_min_params& params) const {
     auto jit = ArgMaxMinKernelBase::GetJitConstants(params);
 
-    jit.AddConstant(MakeJitConstant("OPERATION_NUM", getOperationNumber(params)));
+    if (params.has_dynamic_tensors()) {
+        const std::string gws_0 = "get_global_size(0)";
+        const std::string operation_num_comp = "(GLW_0!=1)";
+        const std::string operation_num = getOperationNumberString(params);
+        jit.AddConstant(MakeJitConstant("GWS_0", gws_0));
+        jit.AddConstant(MakeJitConstant("OPERATION_NUM_COMP", operation_num_comp));
```

Comment:
`const std::string operation_num` is not used anywhere
What is `GLW_0` in `const std::string operation_num_comp`? As of now `#if OPERATION_NUM_COMP` condition will be always true in dynamic cases - is it your intention?

Reply:
`GLW_0` is calculated by `operation_num > 1 ? align(operation_num, 32) : 1` in `SetDefault`. Therefore, if `GWS_0` is not 1, it indicates that `operation_num` is greather than 1. Accordingly, kernel uses `#if OPERATION_NUM_COMP` instead of `#if OPERATION_NUM > 1` to conver both static and dynamic kernel cases.